action.yml

```diff
@@ -14,11 +14,11 @@ inputs:
     default: false
   access_token:
     description: 'Your GitHub Access Token, defaults to: {{ github.token }}'
-    default: '${{ github.token }}'
-    required: true
+    default: ${{ github.token }}
+    required: false
   all_but_latest:
     description: "Cancel all actions but the last one"
     required: false
 runs:
-  using: 'node12'
+  using: 'node14'
```

Comment:
This isn't possible yet but you can thumbs up this issue: https://github.com/actions/runner/issues/772